### PR TITLE
Ensure git hooks are installed as part of rake install

### PR DESCRIPTION
**Pull request by islemaster**
Suggested by @mehalshah in #6423.  This is a little pushy, but I think it's reasonable since it's local to our repo.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -260,7 +260,7 @@ namespace :install do
 
     files.each do |f|
       path = File.expand_path("../tools/hooks/#{f}", __FILE__)
-      system "ln -s #{path} #{git_path}/#{f}"
+      RakeUtils.ln_s path, "#{git_path}/#{f}"
```
Comment (islemaster) [inline Rakefile:280]: This helper has an 'ensure' behavior, so it won't complain if the symlink is already correct.

**Comment by islemaster** [comment]
FYI @wjordan - are you cool with this?

**Comment by mehalshah** [comment]
LGTM - pending will's comments

**Comment by wjordan** [comment]
:+1:

**Comment by laurelfan** [comment]
should this be development only? I could see both ways. And in the case that we do want them on non-development envs, should we install the hooks on staging and levelbuilder?

**Comment by islemaster** [comment]
Nice catch @laurelfan - I could see an argument for having these on staging/levelbuilder, but I don't want them to break any of our automated commit stuff in those environments.  It seems like the conservative thing to do for now is to make this a dev-only change.  I'll add that check.

**Pull request merged**